packages/components/src/components/SquiggleViewer/index.tsx

```diff
@@ -32,37 +32,54 @@ export type SquiggleViewerProps = {
   resultItem: result<SqValue, SqError> | undefined;
   localSettingsEnabled?: boolean;
   editor?: CodeEditorHandle;
+  rootPathOverride?: SqValuePath;
 } & PartialPlaygroundSettings;
 
 const SquiggleViewerOuter = forwardRef<
   SquiggleViewerHandle,
   SquiggleViewerProps
->(function SquiggleViewerOuter({ resultVariables, resultItem }, ref) {
+>(function SquiggleViewerOuter(
+  { resultVariables, resultItem, rootPathOverride },
+  ref
+) {
   const { focused, dispatch, getCalculator } = useViewerContext();
   const unfocus = useUnfocus();
   const focus = useFocus();
 
   const navLinkStyle =
     "text-sm text-slate-500 hover:text-slate-900 hover:underline font-mono cursor-pointer";
 
-  const focusedNavigation = focused && (
+  const isFocusedOnRootPathOverride =
+    focused && rootPathOverride && pathIsEqual(focused, rootPathOverride);
+
+  // If we're focused on the root path override, we need to adjust the focused path accordingly when presenting the navigation, so that it begins with the root path intead. This is a bit confusing.
+  const rootPathFocusedAdjustment: number = rootPathOverride
+    ? rootPathOverride.items.length - 1
+    : 0;
+
+  const focusedNavigation = focused && !isFocusedOnRootPathOverride && (
     <div className="flex items-center mb-3 pl-1">
-      <span onClick={unfocus} className={navLinkStyle}>
-        {focused.root === "bindings" ? "Variables" : focused.root}
-      </span>
+      {!rootPathOverride && (
+        <div>
```

Comment:
This div breaks the layout:

<img width="282" alt="Captura de pantalla 2023-11-12 a la(s) 16 30 07" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/05d32e7d-58e9-4d6e-bd28-afc0d3074cff">

(screenshot from https://squiggle-components-git-modelexpo-935a09-quantified-uncertainty.vercel.app/?path=/story/squiggle-squigglechart--nested)

Needs `flex items-center`, or maybe extract it to `<WithChevronRight>...</WithChevronRight>` to reuse here and for items below.

Reply:
Good point, that needs to be fixed.